development/server/monitor-server-python/Dockerfile

```diff
@@ -5,4 +5,4 @@ EXPOSE  5000
 WORKDIR /monitor-api-python
```

Comment:
What image is this, is it python-base?
Shouldn't we install here gunicorn instead?